app/src/main/java/com/rizadwi/snapsift/usecase/GetCategoriesUseCase.kt

```diff
@@ -0,0 +1,13 @@
+package com.rizadwi.snapsift.usecase
+
+import com.rizadwi.snapsift.common.wrapper.Result
+import com.rizadwi.snapsift.datasource.repository.SourceRepository
+import dagger.hilt.android.scopes.ViewModelScoped
+import javax.inject.Inject
+
+@ViewModelScoped
```

Comment:
why use this annotation here?

Reply:
i forgot to remove it.  i was just experimenting with hilt dagger scope back then. i'll remove that

Reply:
refactored in PR #19